driver/csiplugin/connectors/rest_v2.go

```diff
@@ -1151,6 +1185,33 @@ func (s *spectrumRestV2) DeleteDirectory(filesystemName string, dirName string)
 	return nil
 }
 
+func (s *spectrumRestV2) StatDirectory(filesystemName string, dirName string) (string, error) {
+	glog.V(4).Infof("rest_v2 StatDirectory. filesystem: %s, dir: %s", filesystemName, dirName)
+
+	NdirName := strings.ReplaceAll(dirName, "/", "%2F")
```

Comment:
`NdirName` what does `N` in this variable name denote?

Reply:
Done